packages/react-composites/src/composites/MeetingComposite/ChatButton.tsx

```diff
@@ -15,13 +16,15 @@ export const ChatButton = (props: ControlBarButtonProps): JSX.Element => {
   const strings = { label: props.label, ...props.strings };
 
   return (
-    <ControlBarButton
-      {...props}
-      labelKey={'chatButtonLabelKey'}
-      strings={strings}
-      onRenderOnIcon={props.onRenderOnIcon ?? onRenderOnIcon}
-      onRenderOffIcon={props.onRenderOffIcon ?? onRenderOffIcon}
-      onClick={props.onClick}
-    />
+    <Stack style={{ position: 'relative' }}>
```

Comment:
Add a comment why this is needed
```
// Chat Button may have an optional notification icon attached to it that must be positioned absolute inside the chat button.
// This requires the container of the notification icon to have `position relative`
```

Reply:
use styles or style here also

Reply:
How does the notification icon get passed into here? Would it better to simply have the notification icon defined here and be part of the ChatButton - I think it's a simple enough API for us to easily support and gives chat button a good bit of extra value